ansible/roles/providers/files/DataSourceVMwareGuestInfo.py

```diff
@@ -0,0 +1,765 @@
+# Cloud-Init Datasource for VMware Guestinfo
```

Comment:
copied from upstream to make it work in offline mode. PR description for details.